pkg/cmd/cli/cmd/rollback.go

```diff
@@ -57,14 +60,14 @@ func NewCmdRollback(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
 	}
 
 	cmd := &cobra.Command{
-		Use:     "rollback DEPLOYMENT",
+		Use:     "rollback [dc/]config-name",
```

Comment:
Don't

On Jul 9, 2015, at 10:58 AM, David Eads <notifications@github.com> wrote:

In pkg/cmd/cli/cmd/rollback.go
<https://github.com/openshift/origin/pull/3643#discussion_r34264171>:

> @@ -57,14 +60,14 @@ func NewCmdRollback(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
>  	}
>
>  	cmd := &cobra.Command{
> -		Use:     "rollback DEPLOYMENT",
> +		Use:     "rollback [dc/]config-name",

I like this better, but it breaks existing scripts. @smarterclayton
<https://github.com/smarterclayton> what's our stance on that?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3643/files#r34264171>.